gapic-generator/templates/default/helpers/default_helper.rb

```diff
@@ -39,6 +39,16 @@ def indent_tail input, spacing
     input.lines[0] + indent(input.lines[1..-1].join, spacing)
   end
 
+  def format_number value
+    return value.to_s if value.abs < 10_000
+    str = value.is_a?(Integer) ? value.to_s : BigDecimal(value.to_f.to_s).to_s("F")
```

Comment:
Just to be explicit, do you want to require `bigdecimal` in this file?

Reply:
Oh. Yes, good call. Done.